velero/spectrum-fusion/recipes/no-olm/core/child-cs-odlm-chart-recipe.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: spp-data-protection.isf.ibm.com/v1alpha1
+kind: Recipe
+metadata:
+  labels:
+    dp.isf.ibm.com/parent-recipe: cpfs-parent-recipe
+    dp.isf.ibm.com/parent-recipe-namespace: <parent recipe namespace>
+  name: cs-odlm-child-recipe
+  namespace: <child recipe namespace>
+spec:
+  appType: common-service
+  groups:
+    - includeClusterResources: true
+      includedResourceTypes:
+        - clusterrole
+        - clusterrolebinding
+        - customresourcedefinitions.apiextensions.k8s.io
```

Comment:
cs cr crd labeled as crd so would not be covered here if using the label-common-service.sh script, resulting in a partially failed